test-suite/coq-makefile/local-late-extension/run.sh

```diff
@@ -2,14 +2,13 @@
 
 . ../template/path-init.sh
 
+rm -rf _test
+mkdir _test
+find . -maxdepth 1 -not -name . -not -name _test -exec cp -r '{}' -t _test ';'
+cd _test
+
```

Comment:
I'm a bit confused why we need this

Reply:
it was that or add `printvo` to gitignore somewhere
I feel this way is nicer

Reply:
Alternatively we could pick a filename that's already ignored, like make it output to `$@.log` or something.

Reply:
But also it doesn't seem like `_test` will be ignored?

Reply:
https://github.com/coq/coq/blob/master/test-suite/coq-makefile/.gitignore

Reply:
Ah, I see.  And you think this is nicer than making it output to `printvo.log` (given that .log files are already .gitignored)?

Reply:
Sure.  I've also already approved.  Should I assign as well?